administration/src/cards/extensions/AddressFieldExtensions.ts

```diff
@@ -24,6 +24,8 @@ const getAddressFieldExtension = <T extends AddressFieldExtension>(
   isValid: () => true,
   fromString: (value: string) => ({ [name]: value } as AddressFieldExtensionState<T>),
   toString: (state): string => state[name],
+  fromSerialized: (value: string) => ({ [name]: value } as AddressFieldExtensionState<T>),
```

Comment:
i think you could also predefine `toString()` and `fromString()` and reuse it like in the other extensions

Reply:
Tried to adjust that but couldn't get it work easily. so nevermind.
For me its good to go